database/schema.sql

```diff
@@ -2,6 +2,44 @@ set client_min_messages to warning;
 
 -- DANGER: this is NOT how to do it in the real world.
 -- `drop schema` INSTANTLY ERASES EVERYTHING.
+
 drop schema "public" cascade;
 
 create schema "public";
+
+CREATE TABLE "public"."accounts" (
+	"accountId" serial NOT NULL,
+	"username" TEXT NOT NULL UNIQUE,
+	"hashedPassword" TEXT NOT NULL,
+	CONSTRAINT "accounts_pk" PRIMARY KEY ("accountId")
+) WITH (
+  OIDS=FALSE
+);
+
+CREATE TABLE "public"."favorites" (
+	"accountId" int NOT NULL,
+	"restaurantId" TEXT NOT NULL UNIQUE
+) WITH (
+  OIDS=FALSE
+);
+
+CREATE TABLE "public"."roulette" (
+	"accountId" int NOT NULL,
+	"restaurantId" TEXT NOT NULL UNIQUE
```

Comment:
In each of these, we need to allow different accounts to favorite or add the same restaurants to the roulette. The unique constraint would go on the pair of `"accountId"` and `"restaurantId"`

```sql
UNIQUE ("accountId", "restaurantId")
```

https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-UNIQUE-CONSTRAINTS